.github/scripts/pipeline/template.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
```

Comment:
What is this file for?

Reply:
For attaching any device to VM, virsh needs an .xml file which contains with device's bus, slot and function values.
Template.xml acts as a template for this .xml file. 

Whenever pcie card need to attached to VM, first template.xml file will be loaded with card's bus, slot and function values. And then, virsh attaches that card to VM using this template.xml file.
XML file editing steps are present in attach_cards.sh script

Reply:
Maybe add this as a comment <!-- ...> to the xml file itself?

Reply:
Hi @stsoe 
Comments have been added to xml file